### PR TITLE
Create 2021-February-17.md

### PR DESCRIPTION
**Pull request by ashishvirani7**

### CODE REVIEW

File: releasenotes/2021-February-17.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+10. Transfer interface changes:
+    1. 'transferStateChanged' event is renamed to 'stateChanged'
+11. LocalVideoStream interface chnages:
+    1. getMediaStreamType() has been changed to **getter method** mediaStreamType()
+    2. getSource() has been changed to **getter method** source()
+12. RemoteVideoStream interface changes:
+    1. type property is renamed to mediaStreamType
+
+## Other changes
+1. Call.remoteParticipants type change: RemoteParticipant[] -> ReadonlyArray<RemoteParticipant>
```
Comment (jsaurezlee-msft) [inline releasenotes/2021-February-17.md:44]: Local Video Streams for call were also changed to ReadonlyArray
readonly localVideoStreams: ReadonlyArray<LocalVideoStream>;

File: releasenotes/2021-February-17.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    1. getCameraList() has been changed to getCameras() which is now **async**
+    2. getMicrophoneList() has been changed to getMicrophones() which is now **async**
+    3. getSpeakerList() has been changed to getSpeakers() which is now **async**
+    4. setMicrophone() has been changed to selectMicrophone() which is now **async**
+    5. setSpeaker() has been changed to selectSpeaker() which is now **async**
+    6. getMicrophone() has been changed to readonly **property** selectedMicrophone
+    7. getSpeaker() has been changed to readonly **property** selectedSpeaker
+    8. 'permissionStateChanged' event has been removed
+    8. New event 'selectedMicrophoneChanged' which will be raised when selectedMicrophone is changed
+    9. New event 'selectedSpeakerChanged' which will be raised when selectedSpeaker is changed
```
Comment (jsaurezlee-msft) [inline releasenotes/2021-February-17.md:27]: We have a new property in Device Manager:

readonly isSpeakerSelectionAvailable: boolean;

This tells the user if the host device supports speaker enumeration and selection.

File: releasenotes/2021-February-17.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+This release notes contain new changes for ACS Calling Web (JavaScript) SDK v1.0.0-beta.5
+# Bug fixes
+1. Fixed https://github.com/Azure/Communication/issues/46
+2. Fixed https://github.com/Azure/Communication/issues/47
+3. Fixed https://github.com/Azure/Communication/issues/144
+4. Fixed https://github.com/Azure/Communication/issues/174
```
Comment (chriswhilar) [inline releasenotes/2021-February-17.md:6]: Id say to also give a brief descriptoin

**Review by chriswhilar — APPROVED** [review]

File: releasenotes/2021-February-17.md (hunk 3 above)
Comment (ashishvirani7) [reply releasenotes/2021-February-17.md:9]: Added!

**Pull request merged**